source/PlayerInfo.cpp

```diff
@@ -2016,6 +2018,58 @@ void PlayerInfo::SetTravelDestination(const Planet *planet)
 
 
 
+void PlayerInfo::QueueTeleport(const Planet *teleportPlanet, bool _flagshiponly)
+{
+	teleportDestination = teleportPlanet;
+	flagshiponly = _flagshiponly;
+}
+
+
+
+void PlayerInfo::DoQueuedTeleport()
+{
+	if(teleportDestination && planet)
+	{
+		flagship->SetSystem( teleportDestination->GetSystem());
+		flagship->SetPlanet(teleportDestination);
+		if(!flagshiponly)
+		{
```

Comment:
Pretty sure the braces for the if and the for can be removed

Reply:
done